elastic/kibana
# [Infra] Refactor useSearchSession to useReloadRequestTime for timestamp based refresh

## Summary

Refactoring `useSearchSession` to `useReloadRequestTime` to decouple the usage of search.session for refreshing, avoiding a re-rendering loop issue. This change is then propagated to various Lens charts to make use of that instead of relying on searchSessionId.

Closes #203412

## How to test

- Go to Infrastructure Inventory page in Observability
- Enable Auto-Refresh
- Click on a infra item (host/service/etc) to open the fly-out
- Update the time range in the flyout
- Things should update without spurious refreshes.
